src/NosCore.PathFinder.Gui/GuiWindow.cs

```diff
@@ -51,15 +53,13 @@ public class GuiWindow : GameWindow
         private static readonly ILogger _logger = Logger.GetLoggerConfiguration().CreateLogger();
         private readonly byte _gridsize;
         private readonly Map _map;
+        private readonly DataAccessHelper _dbContextBuilder;
+        private readonly IDao<MapMonsterDto, int> _mapMonsterDao;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_mapMonsterDao' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)